src/gui/tray/activitylistmodel.cpp

```diff
@@ -608,6 +614,12 @@ void ActivityListModel::slotTriggerAction(const int activityIndex, const int act
 
     const auto action = activity._links[actionIndex];
 
+    // TODO this will change with https://github.com/nextcloud/desktop/issues/4159
+    if (action._verb == "WEB" && action._label == tr("View chat")) {
```

Comment:
@camilasan I hope it does get changed. Comparing translated strings doesn't look reliable.

Reply:
Yes.

Reply:
see https://github.com/nextcloud/spreed/issues/7038#issuecomment-1080719853